### PR TITLE
Add Seven Kingdoms: Ancient Adversaries

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/the3dfxdude/7kaa/issues/183
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://github.com/the3dfxdude/7kaa/pull/253

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build com.sevenkfans.SevenKingdomsAncientAdversaries

**Comment by Mailaender** [comment]
bot, build com.sevenkfans.SevenKingdomsAncientAdversaries

**Comment by lennyhans** [comment]
Hey, I have checkout the url with the data-checker, and goes OK

```sh
flatpak run --filesystem=$(pwd) org.flathub.flatpak-external-data-checker $(pwd)/com.sevenkfans.SevenKingdomsAncientAdversaries.yaml
INFO    src.manifest: Checking 3 external data items
INFO    src.manifest: Started check [1/3] git enet/enet.git (from com.sevenkfans.SevenKingdomsAncientAdversaries.yaml)
INFO    src.manifest: Started check [2/3] git 7kaa/7kaa.git (from com.sevenkfans.SevenKingdomsAncientAdversaries.yaml)
INFO    src.manifest: Started check [3/3] extra-data 7kaa/7kaa-music-2.15.zip (from com.sevenkfans.SevenKingdomsAncientAdversaries.yaml)
INFO    src.manifest: Finished check [1/3] git enet/enet.git (from com.sevenkfans.SevenKingdomsAncientAdversaries.yaml)
INFO    src.manifest: Finished check [2/3] git 7kaa/7kaa.git (from com.sevenkfans.SevenKingdomsAncientAdversaries.yaml)
INFO    src.manifest: Finished check [3/3] extra-data 7kaa/7kaa-music-2.15.zip (from com.sevenkfans.SevenKingdomsAncientAdversaries.yaml)
INFO    src.main: Check finished with 0 error(s)
```

So maybe there is another problem?

**Comment by Mailaender** [comment]
Let's try again.

bot, build com.sevenkfans.SevenKingdomsAncientAdversaries

File: 7kaa.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<?xml version='1.0' encoding='utf-8'?>
+<component type="desktop">
+  <id>com.sevenkfans.SevenKingdomsAncientAdversaries</id>
```
Comment (lennyhans) [inline 7kaa.metainfo.xml:3]: I think that keep the acronym would help the seems more like the upstream bin 
```suggestion
  <id>com.sevenkfans.sevenkaa</id>
```

File: 7kaa.desktop [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Seven Kingdoms
+GenericName=Seven Kingdoms
+Comment=Seven Kingdoms: Ancient Adversaries
+Type=Application
+Exec=7kaa
+Icon=com.sevenkfans.SevenKingdomsAncientAdversaries
+Categories=Game;StrategyGame;
+StartupNotify=false
```
Comment (lennyhans) [inline 7kaa.desktop:9]: Add keywords it is helpful also
```suggestion
StartupNotify=false
Keywords=game;strategy;ancient;kingdoms;real-time;trade;diplomacy;espionage;7kaa;
```

File: 7kaa.metainfo.xml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    <release version="v2.14.7" date="2017-06-04" />
+    <release version="v2.14.6" date="2016-09-04" />
+    <release version="v2.14.5" date="2015-05-28" />
+  </releases>
+  <url type="homepage">https://7kfans.com/</url>
+  <url type="bugtracker">https://github.com/the3dfxdude/7kaa/issues</url>
+  <url type="help">https://www.7kfans.com/downloads/7kaa-manual.pdf</url>
+  <url type="vcs-browser">https://github.com/the3dfxdude/7kaa</url>
+  <content_rating type="oars-1.1">
+    <content_attribute id="violence-fantasy">moderate</content_attribute>
```
Comment (lennyhans) [inline 7kaa.metainfo.xml:49]: The game has non-moderated in-game chat
```suggestion
    <content_attribute id="violence-fantasy">moderate</content_attribute>
    <content_attribute id="social-chat">intense</content_attribute>
```

**Review by lennyhans — COMMENTED** [review]
I have tested the build flatpak and works well :ok_hand:, just a few metadata suggestions

**Comment by Mailaender** [comment]
bot, build com.sevenkfans.sevenkaa

File: 7kaa.desktop [hunk 4]
```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=Seven Kingdoms
+GenericName=Seven Kingdoms
+Comment=Seven Kingdoms: Ancient Adversaries
+Type=Application
+Exec=7kaa
+Icon=com.sevenkfans.SevenKingdomsAncientAdversaries
```
Comment (hfiguiere) [inline 7kaa.desktop:7]: gotta need to fix the icon name too.

**Comment by Mailaender** [comment]
bot, build com.sevenkfans.sevenkaa

**Pull request closed without merging**